### DISCUSSION TITLE
Is there a maven package that includes the full packaged mill assembly?

### DISCUSSION THREAD

**Discussion by DamianReeves** [com-lihaoyi/mill#1097] [Q&A] [answered]
Due to restrictions in my firm I can't download releases from GitHub's releases page, but we can download things from maven central.

What is the package that has the complete assembly for mill?

**Comment by lefou** [comment, marked as answer]
The full assembly is packaged and published with artifact name `mill-dev`.

https://repo1.maven.org/maven2/com/lihaoyi/mill-dev_2.13/

**Reply by DamianReeves** [reply]
Thanks, that's what I was looking for.
